bitbots_docs/docs/manual/tutorials/install_software_ros2.rst

```diff
@@ -47,22 +49,9 @@ If you are not already using Ubuntu 22.04, consider installing it on your system
     - Clone the code repository with: ``git clone git@github.com:bit-bots/bitbots_meta.git``
       Confirm the host key by typing ``yes``, if asked.
     - Move into the newly created directory with: ``cd bitbots_meta``
-    - Clone all sub-repositories and other files by running: ``make pull-init``
-- If you want to run the robot's cameras on your system, also run the following command: ``make basler`` Confirm the host key by typing ``yes``, if asked.
-
-**4. Install additional dependencies**
-
-We need to install the requirements of our software. Most of these can be automatically installed
-with ``rosdep``. In the ``bitbots_meta`` folder, simply run ``sudo rosdep init`` followed by ``rosdep update`` and ``rosdep install --rosdistro=iron --from-paths . --ignore-src -y``.
-
-We also need to install some python packages using ``pip``, the python package manager.
-
-- Upgrade python package manager: ``pip3 install pip -U``
-- Optionally if you want you can setup a local venv with: ``python -m venv venv-bitbots && source venv-bitbots/bin/activate``
-  **HINT**: the sourcing of the ``venv`` is required in every newly opened terminal!
-- Install required python packages: ``pip3 install --user -r requirements.txt``
+    - Clone all sub-repositories and other files by running: ``make install`
```

Comment:
I would like to at least note that the basler drivers are optional and that we have them in a private repo. Otherwise external people might trz to pull it and it fails because of it.

Reply:
Good point! Than I will remove the Basler drivers from the install command and include a description about the optional make Basler command (as before)